lib/msal-common/src/client/RefreshTokenClient.ts

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Microsoft Corporation. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import { Configuration } from "../config/Configuration";
+import {BaseClient} from "./BaseClient";
+import {RefreshTokenRequest} from "../request/RefreshTokenRequest";
+import {Authority, NetworkResponse} from "..";
+import {ServerAuthorizationTokenResponse} from "../server/ServerAuthorizationTokenResponse";
+import {RequestParameterBuilder} from "../server/RequestParameterBuilder";
+import {ScopeSet} from "../request/ScopeSet";
+import {GrantType} from "../utils/Constants";
```

Comment:
nit: Can we maintain the spaces in import statements: `import { BaseClient } ... ;` instead of `import {BaseClient} ... ;`

Reply:
Updated.